pepr/report.py

```diff
@@ -19,6 +19,16 @@
         "against machine learning models 2017 IEEE Symposium on Security and Privacy "
         "(SP). IEEE, 2017."
     ),
+    "L2ContrastReductionAttack": (
+        "Reduces the contrast of the input using a perturbation of the given size."
+    ),
+    "VirtualAdversarialAttack": (
+        "Second-order gradient-based attack on the logits. The attack calculate an "
+        "untargeted adversarial perturbation by performing a approximated second order "
+        "optimization step on the KL divergence between the unperturbed predictions "
+        "and the predictions for the adversarial perturbation. This attack was "
+        "originally introduced as the Virtual Adversarial Training method."
+    ),
```

Comment:
We should check wether we can find references to the papers of these attacks.

Reply:
Added in 0e3fac3

Reply:
Look great! 👍